app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/splash_background"
+    android:theme="@style/splashScreenTheme"
+    tools:context=".SplashScreenActivity">
+
+    <ImageView
+        android:id="@+id/splash_screen_title"
+        android:layout_width="@dimen/splash_title_size"
+        android:layout_height="@dimen/splash_title_size"
+        android:layout_marginStart="@dimen/splash_title_margin"
+        android:layout_marginTop="@dimen/splash_title_margin"
+        android:src="@drawable/ic_splash_title"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/splash_screen_wolf"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/splash_wolf_height"
```

Comment:
You don't really need to set height to a hardcoded value. Won't `wrap_content` with constraints and android:scaleType do the job?